packages/duckiebot_interface/launch/all_drivers.launch

```diff
@@ -3,64 +3,70 @@
     <arg name="param_file_name" default="default" doc="Specify a param file. ex:default." />
     <arg name="robot_type" default="duckiebot" doc="The type of the robot"/>
 
-    <!-- Joystick: reads from a physical joystick and produces ROS messages -->
-    <include file="$(find joystick)/launch/joystick_node.launch">
-        <arg name="veh" value="$(arg veh)"/>
-        <arg name="param_file_name" value="$(arg param_file_name)"/>
-    </include>
-
     <!-- Camera Node: reads frames from a physical camera and produces ROS messages -->
     <include file="$(find camera_driver)/launch/camera_node.launch">
         <arg name="veh" value="$(arg veh)"/>
-        <arg name="param_file_name" value="$(arg param_file_name)"/>
-    </include>
-
-    <!-- Wheel Drivers: reads ROS messages and spins the motors -->
-    <include file="$(find wheels_driver)/launch/wheels_driver_node.launch">
-        <arg name="veh" value="$(arg veh)"/>
-        <arg name="param_file_name" value="$(arg param_file_name)"/>
+        <arg name="param_file_name" value="$(arg robot_type)"/>
     </include>
 
     <!-- LED Emitter: reads ROS messages and commands the LEDs -->
-    <include file="$(find led_emitter)/launch/led_emitter_node.launch">
-        <arg name="veh" value="$(arg veh)"/>
-    </include>
+    <group unless="$(eval arg('robot_type') == 'watchtower')">
```

Comment:
@AmaurX why is this on for watchtower but not traffic light?

Reply:
because the traffic light have leds

Reply:
@AleksandarPetrov basically the default is to have leds, only the watchtowers don't have them

Reply:
it's an `unless` statement. I think it is less dangerous that specifying each type that should have a led launch

Reply:
@AmaurX did you know that starting from Kinetic, `$(eval EXP)` in roslaunch can take any valid python2 expression? So I guess you could try with OR as well.